macros/psng_manual_change.ngc

```diff
@@ -17,11 +19,12 @@ O100     return [999]
 O100 endif
 
 ;first go up
+F#<_ini[TOOLSENSOR]RAPID_SPEED>
 G90
-G53 G0 Z[#<_ini[AXIS_Z]MAX_LIMIT>-0.1]
+G53 G1 Z[#<_ini[AXIS_Z]MAX_LIMIT>-1]
 ; then move to change position
-G53 G0 X[#<_ini[CHANGE_POSITION]X>] Y[#<_ini[CHANGE_POSITION]Y>]
-G53 G0 Z[#<_ini[CHANGE_POSITION]Z>]
+G53 G1 X[#<_ini[CHANGE_POSITION]X>] Y[#<_ini[CHANGE_POSITION]Y>]
+G53 G1 Z[#<_ini[CHANGE_POSITION]Z>]
```

Comment:
Switching to G1 means the rapid override in the UI no longer works, and at least with my configuration, will make the probe maneuvers far slower. Why would G1 be preferred here?